### DISCUSSION TITLE
History versions of a note automatically generated

### DISCUSSION THREAD

**Discussion by vastlander** [glushchenko/fsnotes#1567] [Q&A]
Platform: iOS 16.5.1
FSNotes Version: 6.0.3 Build 265

1. When I created a note and edited it in my folder, several history versions of this note were always automatically generated, and I had to manually delete these versions every time. how to solve it?

2. When I choose to create and edit a note in Inbox instead, a large number of empty notes are generated, I have to quit the app to make them disappear. Is there a better way to solve it?

**Comment by glushchenko** [comment]
I dont know how to reproduce it. Did you use git?

**Reply by vastlander** [reply]
I didn’t use git.
